tests/unit/demography/test_community.py

```diff
@@ -113,26 +131,48 @@ def test_Cohorts(args, outcome, excep_message):
     assert str(excep.value) == excep_message
 
 
+def test_Cohorts_duplicate_id_detection():
+    """Test the property checking for duplicate cohort ids."""
+
+    from pyrealm.demography.community import Cohorts
+
+    # Create and instances to modify using methods
+    cohorts = Cohorts(
+        pft_names=np.array(["broadleaf", "conifer"]),
+        n_individuals=np.array([6, 1]),
+        dbh_values=np.array([0.2, 0.5]),
+    )
+
+    with pytest.raises(ValueError):
+        _ = cohorts.add_cohort_data(new_data=cohorts)
+
+
 def test_Cohorts_CohortMethods():
     """Test the inherited CohortMethods methods."""
 
     from pyrealm.demography.community import Cohorts
 
-    # Create and instance to modify using methods
+    # Create and instances to modify using methods
```

Comment:
```suggestion
    # Create instances to modify using methods
```
I think this was an existing typo that was missed